components/viaduct/src/error.rs

```diff
@@ -4,27 +4,27 @@
 
 #[derive(Debug, thiserror::Error)]
 pub enum Error {
-    #[error("Illegal characters in request header '{0}'")]
+    #[error("[no-sentry] Illegal characters in request header '{0}'")]
```

Comment:
Honestly not all of these need to be no-sentry.

I'm not sure which we're seeing in sentry. I guess we might want reporting on anything other than BackendError and NetworkError?

That said, the safe thing seemed to be to silence all of them